salt/_modules/metalk8s_kubernetes.py

```diff
@@ -139,6 +139,9 @@ def method(manifest=None, name=None, kind=None, apiVersion=None,
             manifest.setdefault('metadata', {}).setdefault('labels', {})[
                 'metalk8s.scality.com/version'
             ] = match.group('version') if match else "unknown"
+            manifest['metadata']['labels']['app.kubernetes.io/managed-by'] = \
+                'salt'
+            manifest['metadata']['labels']['heritage'] = 'salt'
```

Comment:
:+1:

Reply:
Right now for things like coredns we put "metalk8s" as heritage

Reply:
Also I think we shouldn't override these value if they are already in the manifest (and indeed it's the same for version just above)